lighthouse-core/report/html/report-styles.css

```diff
@@ -1372,6 +1372,11 @@
   object-fit: contain;
 }
 
+a.lh-anchor {
+  color: inherit;
```

Comment:
so the only indication will be cursor?

Reply:
yes - I think it's fair to say it's surprising that a URL isn't clickable. So is visually denoting that it is redundant? I didn't want to change anything visually if we don't need to.

Reply:
I don't think we want to do this styling for all `link` audit-details as well. We already have the class for text urls, what about `.lh-text__url > a` (and removing the `lh-link` class from the `createElement` call)? crc nodes would also need to be selected but having that explicit seems like a positive

Reply:
done